streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -711,12 +710,7 @@ private void checkBrokerVersionCompatibility() throws StreamsException {
 
         client.checkBrokerCompatibility(EXACTLY_ONCE.equals(config.getString(PROCESSING_GUARANTEE_CONFIG)));
 
-        try {
-            client.close();
```

Comment:
Could you elaborate a bit on this change?

Reply:
Our internal client never throws on `close()` (cf https://github.com/apache/kafka/pull/4128/files#diff-4320e27e72244cae71428533cf3582efR173) -- no reason to catch anything.